lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+msgid "Old settings found. Default PID, Esteps etc. will be set."
+msgstr "Gamla inst\xe1llningar hittades. Standard PID, Esteps etc. kommer att st\xe1llas in."
+
+# MSG_REMOVE_TEST_PRINT c=20 r=4
+#: ultralcd.cpp:4958
+msgid "Now remove the test print from steel sheet."
+msgstr "Ta nu bort testutskriften fran metallskivan."
+
+# MSG_NOZZLE_FAN c=10
+#: ultralcd.cpp:1446
+msgid "Nozzle FAN"
+msgstr "Muns fl\xe1kt"
+
+# MSG_PAUSE_PRINT c=18
+#: messages.c:74
+msgid "Pause print"
+msgstr "Pausa utskrift"
+
+# MSG_PID_RUNNING c=20
+#: ultralcd.cpp:1327
+msgid "PID cal."
+msgstr "PID kalibrering."
+
+# MSG_PID_FINISHED c=20
+#: ultralcd.cpp:1333
+msgid "PID cal. finished"
+msgstr "PID kalibrering klar"
+
+# MSG_PID_EXTRUDER c=17
+#: ultralcd.cpp:5799
+msgid "PID calibration"
+msgstr "PID kalibrering"
+
+# MSG_PINDA_PREHEAT c=20
+#: ultralcd.cpp:683
+msgid "PINDA Heating"
+msgstr "PINDA uppv\xe1rmning"
+
+# MSG_PAPER c=20 r=10
+#: messages.c:73
+msgid "Place a sheet of paper under the nozzle during the calibration of first 4 points. If the nozzle catches the paper, power off the printer immediately."
+msgstr "Placera ett pappersark under munstycket under kalibreringen av de f\xefrsta 4 punkterna. St\xe1ng av skrivaren omedelbart om munstycket r\xefr vid pappret.."
+
+# MSG_WIZARD_CLEAN_HEATBED c=20 r=8
+#: ultralcd.cpp:5024
+msgid "Please clean heatbed and then press the knob."
+msgstr "Reng\xefr b\xe1dden och tryck sedan pa knappen."
+
+# MSG_CONFIRM_NOZZLE_CLEAN c=20 r=8
+#: messages.c:26
+msgid "Please clean the nozzle for calibration. Click when done."
+msgstr "Reng\xefr munstycket f\xefr kalibrering. Klicka n\xe1r du \xe1r klar."
+
+# MSG_SELFTEST_PLEASECHECK c=20
+#: ultralcd.cpp:8122
+msgid "Please check:"
+msgstr "Kontrollera:"
+
+# MSG_WIZARD_CALIBRATION_FAILED c=20 r=8
+#: messages.c:117
+msgid "Please check our handbook and fix the problem. Then resume the Wizard by rebooting the printer."
+msgstr "Se var handbok och atg\xe1rda problemet. Ateruppta sedan guiden genom att starta om skrivaren."
+
+# MSG_CHECK_IDLER c=20 r=5
+#: Marlin_main.cpp:3798
+msgid "Please open idler and remove filament manually."
+msgstr "\xefppna idler och ta bort filamentet manuellt."
+
+# MSG_PLACE_STEEL_SHEET c=20 r=5
+#: messages.c:75
+msgid "Please place steel sheet on heatbed."
+msgstr "Placera metallskiva pa v\xe1rmeb\xe1dden."
+
+# MSG_PRESS_TO_UNLOAD c=20 r=4
+#: messages.c:79
+msgid "Please press the knob to unload filament"
+msgstr "V\xe1nligen tryck pa knappen f\xefr att mata ut filament"
+
+# MSG_PULL_OUT_FILAMENT c=20 r=4
+#: messages.c:81
+msgid "Please pull out filament immediately"
+msgstr "V\xe1nligen ta ut filamentet omedelbart"
+
+# MSG_EJECT_REMOVE c=20 r=4
+#: mmu.cpp:1421
+msgid "Please remove filament and then press the knob."
+msgstr "Ta bort filamentet och tryck sedan pa knappen."
+
+# MSG_REMOVE_STEEL_SHEET c=20 r=4
+#: messages.c:84
+msgid "Please remove steel sheet from heatbed."
+msgstr "Ta bort metallskivan fran v\xe1rmeb\xe1dden."
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.